### ISSUE TITLE
Handlebars adds breaking spaces after HTML attribute keys?

### ISSUE THREAD

**Issue by janwerkhoven** [beautify-web/js-beautify#872]
This bug forces me to turn of "beautify on save" for .HBS files...

```
// Expected
<form onsubmit={{action 'submit'}}>
  {{hd-email-input}}
</form>

// What JS Beautify does...
<form onsubmit= {{action 'submit'}}>
  {{hd-email-input}}
</form>
```

It added a space after `onsubmit`, which literally breaks the code, not just the formatting.

[Debug log Gist](https://gist.github.com/janwerkhoven/bc550b68f3103efd3c03)

[Atom Beautify issue](https://github.com/Glavin001/atom-beautify/issues/800)

**Comment by bitwiseman** [comment]
Related to #660

**Comment by MacKLess** [comment]
This is fixed as of v1.8.1
